manifests/base/grafana/dash-k8s-apiserver.yaml

```diff
@@ -977,14 +981,16 @@ data:
       "refresh": "1m",
       "schemaVersion": 27,
       "style": "dark",
-      "tags": ["kubernetes-mixin"],
+      "tags": [
+        "kubernetes-mixin"
+      ],
       "templating": {
         "list": [
           {
             "current": {
               "selected": false,
-              "text": "Observatorium",
-              "value": "Observatorium"
+              "text": "Prometheus",
```

Comment:
why change from Observatorium to Prometheus?

Reply:
Probably changed by default from development instance.